test/controllers/HistoricDateRequestPageControllerSpec.scala

```diff
@@ -17,18 +17,23 @@
 package controllers
 
 import base.SpecBase
-import models._
+import config.FrontendAppConfig
+import forms.HistoricDateRequestPageFormProvider
```

Comment:
We can remove this unused import and the same for line 31.

Reply:
Done